app/controllers/application_controller.rb

```diff
@@ -80,6 +80,10 @@ def failed_update_error(obj, obj_name)
   def failed_destroy_error(obj, obj_name)
     "#{_('Could not delete the %{o}.') % {o: obj_name}} #{errors_to_s(obj)}"
   end
+  
+  def success_message(obj_name, action)
```

Comment:
Did some more reading on this and I can't see a use case where we would want to generate these standard error/success messages outside of a controller. Most of the controllers call these methods so I think it makes sense to leave them in the base application_controller like we've done with the auth, previous path and localization stuff.